### ISSUE TITLE
Planemo 0.22.1 still has problems with Galaxy 16.01.

### ISSUE THREAD

**Issue by jmchilton** [galaxyproject/galaxy#1503]
Working on it now...

**Comment by bgruening** [comment]
With the patch about switching sh to bash, latest 16.01 I need to do the following to get it work:

``` bash
cd $GALAXY_ROOT
./run.sh
. .venv/bin/activate
pip install -r lib/galaxy/dependencies/dev-requirements.txt
```

**Comment by jmchilton** [comment]
Opps - I created this issue on the wrong repo :disappointed:, sorry for the noise all.

@bgruening I am thinking https://github.com/galaxyproject/planemo/commit/1ba8ccadd7f4c66001a9a6ed99ef29a1a649af22 will solve this problem.

Update: This commit as been released as part of planemo 0.22.2 to PyPI and homebrew tap.
